### PR TITLE
Fix some forkchoice typos

### PR DESCRIPTION
**Pull request by terencechain**

### CODE REVIEW

File: specs/_features/epbs/fork-choice.md [hunk 1]
```diff
@@ -221,20 +222,19 @@ def is_payload_present(store: Store, beacon_block_root: Root) -> bool:
...
-        return [root, store.is_payload_present(root)]
+        return (root, store.is_payload_present(root))
 
     assert block.slot > slot
     parent = store.blocks[block.parent_root]
     if parent.slot > slot:
         return get_ancestor(store, block.parent_root, slot)
-    if block.body.signed_execution_payload_header.message.parent_block_hash == 
-        parent.body.signed_execution_payload_header.message.block_hash:
+    if block.body.signed_execution_payload_header.message.parent_block_hash == parent.body.signed_execution_payload_header.message.block_hash:
```
Comment (terencechain) [inline specs/_features/epbs/fork-choice.md:237]: I think this needs to be one line or python test generation will fail

File: specs/_features/epbs/fork-choice.md [hunk 2]
```diff
@@ -150,7 +150,7 @@ def block_for_inclusion_list(store: Store, inclusion_list: InclusionList) -> Opt
     
     blocks = blocks_for_slot(store, summary.slot)
     for block in blocks:
-        if block.slot == summary.slot and block.proposer_index == summary.proposer_index and block.signed_execution_payload_header.message.parent_block_hash == parent_hash:
```
Comment (terencechain) [inline specs/_features/epbs/fork-choice.md:153]: I dont think you need the slot check. Please correct me if I'm wrong here. `blocks_for_slot(store, summary.slot)` already covers that

Comment (potuz) [reply specs/_features/epbs/fork-choice.md:153]: Nice one! thanks

File: specs/_features/epbs/fork-choice.md [hunk 3]
```diff
@@ -126,7 +126,7 @@ def verify_inclusion_list(state: BeaconState, block: BeaconBlock, inclusion_list
     return execution_engine.notify_new_inclusion_list(NewInclusionListRequest(
             inclusion_list=inclusion_list.transactions, 
             summary=summary,
-            parent_block_hash = state.latest_execution_payload_header.block_hash))
+            parent_block_hash = state.execution_payload_header.block_hash))
```
Comment (potuz) [inline specs/_features/epbs/fork-choice.md]:
```suggestion
            parent_block_hash = state.latest_block_hash))
```

**Pull request merged**